README.md

```diff
@@ -35,7 +35,7 @@ buildConfig({
 })
 ```
 
-To add a documentation UI, use the `swaggerUI`, `rapidoc` or `redoc` plugins, respectively:
+To add a documentation UI, use the `scalar`, `swaggerUI`, `rapidoc` or `redoc` plugins, respectively:
```

Comment:
Would you mind making this a list instead and perhaps add links to the projects? It's getting kinda crowded in there :grin: